core/src/main/scala/ai/h2o/sparkling/backend/external/ExternalBackendUtils.scala

```diff
@@ -59,9 +60,8 @@ private[backend] trait ExternalBackendUtils extends SharedBackendUtils {
     }
   }
 
-
-  protected[backend] def startH2OClient(hc: H2OContext, nodes: Array[NodeDesc]): Unit = {
-    val conf = hc.getConf
+  protected[backend] def startH2OClient(hc: H2OContext, conf: H2OConf, nodes: Array[NodeDesc]): Unit = {
```

Comment:
Why do we need to pass `H2OConf` separately?

Reply:
Good question -> `hc.getConf` returns clone of `conf`, but the `conf` passed here is still accepts changes and is stored inside H2OContext after all the initialization

Reply:
So we still need to update the right conf. Ideal would be to have val conf on `H2OContext` visible just in `ai.h2o.sparkling` but that we can do after we have all classes in the same package

Reply:
Ok, cool. Thanks for your explanation!